www/gatsby-node.js

```diff
@@ -21,6 +21,22 @@ exports.onCreateWebpackConfig = ({ actions }) => {
     actions.setWebpackConfig({
         resolve: {
             modules: [path.resolve(__dirname, 'src'), 'node_modules'],
+            alias: {
+                // Use Thumbprint React and SCSS source files when `import`'ing
+                // them in the documentation. By default, Webpack will use the
+                // value of the `main` or `module` field in the package's
+                // `package.json` file. This makes it difficult to develop
+                // locally because it'd require running re-compiling the code
```

Comment:
"because it'd require running re-compiling the code" do you mean "it would require re-compiling..." ?

Reply:
Yup, good catch. Did the comment make sense otherwise?

Reply:
Yup